go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/onsi/gomega v1.7.1 // indirect
 	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
 	github.com/pingcap/errors v0.11.4
-	github.com/pingcap/kvproto v0.0.0-20191210040729-c23886becb54
+	github.com/pingcap/kvproto v0.0.0-20191211032946-5dbce7e7b868
```

Comment:
Please use kvproto@release-3.1, @MyonKeminta is working on cherry-pick.

Reply:
Cherry-pick is here: https://github.com/pingcap/kvproto/pull/520